opensrp-app/src/main/java/org/smartregister/sync/ClientProcessorForJava.java

```diff
@@ -663,7 +658,7 @@ public void updateClientDetailsTable(Event event, Client client) {
             if (StringUtils.isNotBlank(gender)) {
                 map.put(GENDER, gender);
             }
-        } catch (Exception e) {
+        } catch (NullPointerException e) {
```

Comment:
Is this the only the exception that can occur

Reply:
yeah i think so